.github/workflows/test_runner.yml

```diff
@@ -77,6 +77,9 @@ jobs:
       # Store the sha for checkout so we can easily use it later.  For safe
       # events, this will be blank and use the defaults.
       checkout-sha: ${{ steps.safe-checkout.outputs.sha }}
+      # Stores a string denoting whether this is a presubmit or continuous
+      # run. This helps us determine which tests to block on.
+      test-type: ${{ (github.event_name == 'pull_request' || github.event_name == 'pull_request_target') && 'presubmit' || 'continuous' }} 
```

Comment:
It feels a bit awkward to put this in a "Check for Safety" job.  Should we rename this?